tests/sentry/api/endpoints/test_project_codeowners_details.py

```diff
@@ -107,3 +110,38 @@ def test_codeowners_email_update(self):
             response = self.client.put(self.url, data)
         assert response.status_code == 200
         assert response.data["raw"] == "# cool stuff comment\n*.js admin@sentry.io\n# good comment"
+
+    def test_codeowners_max_raw_length(self):
+        with mock.patch("sentry.api.endpoints.project_codeowners.MAX_RAW_LENGTH", 56):
```

Comment:
i'm guessing the `56` came from the length of the `data["raw"]`? If so could you do `len[data["raw"]] -1` That way changing the test string wouldn't break the test.